README.md

```diff
@@ -20,9 +20,7 @@ WOLMO is a an opinionated framework designed to reduce the time taken to develop
 WOLMO is made up of several modules, each one supporting a different role in the production of modern apps. The following are the current modules available for the WOLMO framework.
 
 * **CORE:** This module provides standalone base classes and utilities of the framework. As a general rule, this module will be required in any WOLMO-based project. CORE is also a generally required dependency of other modules.
-* **NETWORKING:** This module provides offline capabilities and abstracts the network layer of the rest of the project.
-* **AUTH:** Provides utilities for accounts, login and signup features.
-* **MAPS:** Google Maps helpers classes and views.
+* **COMPOSE:** This module provides utilities and abstractions to help the development of Android Apps that uses the Compose Framework. 
```

Comment:
We should add Wolmo Testing here too! Also, maybe it would be a good idea to include a link to the repo.